include/uxr/agent/transport/serial/baud_rate_table_linux.h

```diff
@@ -100,6 +100,7 @@ speed_t getBaudRate(const char* baudrate_str)
     {
         rv = B230400;
     }
+#ifndef __APPLE__
```

Comment:
This could lead to a bug, as if you set a baudrate to `"460800"`, the last else will set the baudrate anyway.

You could add something like this just before the return:
```
#ifdef __APPLE__
      if (230400 < rv)
      {
          rv = 230400;
      }
#endif // __APPLE__
```

Reply:
I'm actually not sure if this would lead to a bug. It's just that on macOS in the `termios.h` header, the standard speeds are only defined up to 230400. It looks like that for other operating systems (ex. [IBM AIX](https://www.ibm.com/docs/en/aix/7.2?topic=files-termiosh-file)), the standard speeds are capped even lower at 38400.  Hence, setting a custom higher baud rate might be still possible. My guess would be that if it is possible to set a custom baud rate in Linux, it should also be possible in macOS or any other Unix based operating system.